cmd/thanos/store.go

```diff
@@ -42,6 +42,9 @@ func registerStore(m map[string]setupFunc, app *kingpin.Application, name string
 	indexCacheSize := cmd.Flag("index-cache-size", "Number of data records that are kept in the index cache.").
 		Default("100000").Int()
 
+	chunkPoolSize := cmd.Flag("chunk-pool-size", "Maximum size of concurrently allocatble bytes for chunks.").
+		Default("2GB").Bytes()
```

Comment:
uhm.. Is that not too small? I guess it's just a reasonable default for small setups

Reply:
Yea, I think this flag always needs to be adjusted in some way and 2GB is a good default for minimal "get-it-running" deployments.